docs/components/StatusIndicator/StatusIndicator.stories.mdx

```diff
@@ -0,0 +1,48 @@
+import { Canvas, Meta } from "@storybook/addon-docs";
+import { StatusIndicator } from "@jobber/components/StatusIndicator";
+import { Content } from "@jobber/components/Content";
+
+<Meta title="Components/Status and Feedback/StatusIndicator/Docs" />
+
+# Status Indicator
+
+The StatusIndicator component is a purely visual component that indicates a
+status (e.g., "Active", "Overdue").
+
+## Usage guidelines
```

Comment:
I think we should add another sentence in between about "wanting a label? use StatusLabel"